core/src/view/view.cpp

```diff
@@ -317,11 +343,54 @@ void View::updateTiles() {
     glm::dvec2 b = (glm::dvec2(viewBR.x + m_pos.x, viewBR.y + m_pos.y) - tileSpaceOrigin) * tileSpaceAxes;
     glm::dvec2 c = (glm::dvec2(viewTR.x + m_pos.x, viewTR.y + m_pos.y) - tileSpaceOrigin) * tileSpaceAxes;
     glm::dvec2 d = (glm::dvec2(viewTL.x + m_pos.x, viewTL.y + m_pos.y) - tileSpaceOrigin) * tileSpaceAxes;
+
+    // Location of the view center in tile space
+    glm::dvec2 e = (glm::dvec2(m_pos.x, m_pos.y) - tileSpaceOrigin) * tileSpaceAxes;
+    
+    // Determine zoom reduction for tiles far from the center of view
+    double tilesAtFullZoom = std::max(m_width, m_height) * invTileSize * 0.5;
+    double viewCenterX = (m_pos.x + hc) * invTileSize;
+    double viewCenterY = (m_pos.y - hc) * -invTileSize;
+    
+    int x_l_pos[MAX_LOD] = { 0 };
```

Comment:
Logic is great!!

Just a small concern with the naming of these lod threshold maps. Can you name these appropriately?